### ISSUE TITLE
EDD Form, insufficient color contrast

### ISSUE THREAD

**Issue by wlla** [NYPL/design-toolkit#181]
Red used for "required" tag fails (foreground #ff0000, background #fff)

**Comment by ricardoom** [comment]
not sure which red is being used, but it should not be #ff0000. from the TK, the reds used have been tried and true on the a11y front...

here are all the colores in question: 
http://contrast-grid.eightshapes.com/?background-colors=&foreground-colors=%23FFFFFF%0D%0A%23F6D6D8%0D%0A%23D0343A%0D%0A%2397272C%0D%0A%23FF0000%0D%0A%23000%0D%0A&es-color-form__tile-size=compact

**Comment by EdwinGuzman** [comment]
This may be an old ticket that has since been resolved? The required red on https://qa-www.nypl.org/research/collections/shared-collection-catalog/hold/request/b13518955-i16541459/edd is not #ff0000.
